environment.yml

```diff
@@ -39,7 +39,7 @@ dependencies:
         - crds
         - exotic-ld==3 # Lower limit needed for updated JWST sensitivity files, upper limit needed for breaking changes
         - image_registration@git+https://github.com/keflavich/image_registration@master # Need GitHub version to avoid np.float issue
-        - jwst==1.11.4
+        - jwst==1.12.2
```

Comment:
Why are we updating this only here?

Reply:
It wasn't updated here.  Now it matches `setup.cfg`.